h/templates/client/annotation.html

```diff
@@ -109,7 +109,16 @@
             ng-click="vm.revert()"
             ><i class="h-icon-x btn-icon"></i> Cancel</button>
   </div>
-</footer>
+</div>
+
+<div class="annotation-section annotation-license" ng-if="vm.editing">
+  <a href="http://creativecommons.org/publicdomain/zero/1.0/"
+     title="View more information about the Creative Commons Public Domain license"
+     target="_blank">
+    <i class="h-icon-cc-logo"></i><i class="h-icon-cc-zero"></i>
+    Annotations can be freely reused by anyone for any purpose.
+  </a>
+</div>
```

Comment:
@csillag this is another place that should be "informed" by a configuration file (not hard coded). If you're still thinking on this problem. :wink: